orca-igor/src/test/groovy/com/netflix/spinnaker/orca/igor/tasks/GetBuildPropertiesTaskSpec.groovy

```diff
@@ -146,6 +146,20 @@ class GetBuildPropertiesTaskSpec extends Specification {
     e.message == "Expected properties file $PROPERTY_FILE but it was either missing, empty or contained invalid syntax"
   }
 
+  def "does not fail stage even if properties are not returned from travis and build passed"() {
+    given:
+    def stage = createStage(PROPERTY_FILE, "travis-$MASTER")
+
+    and:
+    buildService.getPropertyFile(BUILD_NUMBER, PROPERTY_FILE, MASTER, JOB) >> [:]
```

Comment:
This should reference `travis-$MASTER` as well. The test works because Spock returns an empty list by default for mocks.